### PR TITLE
Add Html target attribute 

### PR DESCRIPTION
**Pull request by Shmew**

### CODE REVIEW

File: Feliz/Properties.fs [hunk 1]
```diff
@@ -1137,6 +1137,19 @@ module prop =
...
+        /// Opens the linked document in a new window or tab.
+        static member inline blank = Interop.mkAttr "target" "_blank"
+        /// Opens the linked document in the same frame as it was clicked (this is default).
+        static member inline self = Interop.mkAttr "target" "_self"
+        /// Opens the linked document in the parent frame.
+        static member inline parent = Interop.mkAttr "target" "_parent"
+        /// Opens the linked document in the full body of the window.
+        static member inline top = Interop.mkAttr "target" "_top"
+        /// Opens the linked document in a named frame.
+        static member inline frameName (value: string) = Interop.mkAttr "target" value
```
Comment (cmeeren) [inline Feliz/Properties.fs]: This should be just a normal prop in the `prop` type, taking a `frameName: string` parameter.

Comment (Shmew) [reply Feliz/Properties.fs]: Wouldn't that make it more ambiguous? They'd be doing `prop.target.prop "someFrameName"` vs `prop.target.frameName "someFrameName"`

Comment (cmeeren) [reply Feliz/Properties.fs]: No, I'm talking about this:

```f#
prop.target "someFrameName"  // normal prop
prop.target.blank  // enum prop
```

Comment (Shmew) [reply Feliz/Properties.fs]: Oh, of course! Yes, that absolutely is more correct. Thank you I'll fix that.

**Comment by Zaid-Ajaj** [comment]
Thanks @Shmew :heart:

**Pull request merged**